pkg/cloud/services/ec2/instances.go

```diff
@@ -181,10 +181,13 @@ func (s *Service) CreateInstance(scope *scope.MachineScope) (*v1alpha2.Instance,
 	s.scope.V(2).Info("Running instance", "machine-role", scope.Role())
 	out, err := s.runInstance(scope.Role(), input)
 	if err != nil {
+		if !awserrors.IsFailedDependency(errors.Cause(err)) {
```

Comment:
Do we want to also bring the comment over for why we are only conditionally emitting the event.

Reply:
yes we do. This bit was slightly tricky due to the v1a2 differences and I must have missed it, good catch